guides/dev-environment/README.md

```diff
@@ -15,7 +15,7 @@ Verify its installed by running:
 ~$ pip --version
 ```
 
-## Step 2: Installing virtualenv
+## Step 2: Setting Up virtualenv
```

Comment:
This section explains how to install virtualenv so I guess that "Installing" is more suitable.